composer.json

```diff
@@ -7,14 +7,14 @@
     "prefer-stable": true,
     "require": {
         "php": ">=8.0",
-        "drupal/core": "^9.4 || ^10",
-        "drupal/graphql": "^4.5"
+        "drupal/core": "^10",
+        "drupal/graphql_core_schema": "dev-1.0.x"
```

Comment:
I get this error running composer update:
![image](https://github.com/openeuropa/oe_graphql/assets/5982573/c589717d-ef11-4e75-a68d-3a9d333eff90)
I had to run "composer update graphql_core_schema" to install it

Reply:
Ok but this has nothing to do with the current PR, that's an outdated composer.lock on your local copy. Can you install the project starting from a clean slate?

Reply:
yes, i had to delete all and i worked